lib/jest-helpers.js

```diff
@@ -83,6 +83,53 @@ function renderTemplate (params = {}, blocks = {}) {
   return cheerio.load(output)
 }
 
+/**
+ * Render and initialise a component within test boilerplate
+ *
+ * Renders a component's Nunjucks macro with the given params, injects it into
+ * the test boilerplate page, then initialises it with either:
+ *
+ * - the provided JavaScript configuration, used to instanciate the relevant
+ *   component class
+ * - a custom initialiser function (that'll let you run arbitrary code, like
+ *   `initAll` in the browser)
```

Comment:
this is a bit unclear: does the custom initialiser function let me run arbitrary code:

* for example, `initAll`
* that looks like `initAll`, but in the browser

Or is the custom initialiser function similar to `initAll`, but in the browser?